src/graphql/admin/mongoose_graphql_vcard_admin_mutation.erl

```diff
@@ -0,0 +1,23 @@
+-module(mongoose_graphql_vcard_admin_mutation).
+-behaviour(mongoose_graphql).
+
+-include("mod_vcard.hrl").
+
+-export([execute/4]).
+
+-import(mongoose_graphql_helper, [make_error/2, format_result/2, null_to_default/2]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+-include("mongoose.hrl").
+-include("jlib.hrl").
+
+execute(_Ctx, vcard, <<"setVcard">>, #{<<"user">> := CallerJID, <<"vcard">> := VCARD}) ->
+    case mod_vcard_api:set_vcard(CallerJID, VCARD) of
+        {ok, _} = Vcard -> Vcard;
+        {error, not_found} ->
+            make_error({error, "User does not exist"}, #{user => CallerJID});
```

Comment:
The API modules that I did was intended to return result in the format: {ok, Value/Msg} or {ErrorCode, ErrorMsg}, so in this case, `mod_vcard_api` should return {not_found, "User does not exist"} or {internal, "Internal server error"}

Reply:
After that change, only one make_error should handle each error :)